### DISCUSSION TITLE
How to disable the wait time after hitting `esc` key multiple times? It's annoying the hell out of me

### DISCUSSION THREAD

**Discussion by ElSamhaa** [sayanarijit/xplr#684] [Q&A]
Great project btw 🙂

**Comment by sayanarijit** [comment]
Shouldn't happen... Try in another system?

**Reply by ElSamhaa** [reply]
If I press a key right after pressing `esc`, xplr will do nothing and will instead show a log message `WARNING) key map not found.`

**Reply by ElSamhaa** [reply]
So, I have to wait to be able to press any key, e.g navigate or manipulate a file or directory

**Reply by sayanarijit** [reply]
I can't reproduce it in my system. Esc and right key work as they should.

**Reply by ElSamhaa** [reply]
OK. trying under a vm with no custom config there.

For now, this is my config

```yaml
version = '0.21.3'

xplr.config.general.selection_ui.style = {
  add_modifiers = { "CrossedOut", "Reversed" },
}

xplr.config.general.focus_selection_ui.style = {
  add_modifiers = { "Bold", "CrossedOut", "Reversed" },
}

xplr.config.modes.custom.Z = {
  name = "Z",
  key_bindings = {
    on_key = {
      Z = {
        messages = {
          "PrintPwdAndQuit",
        },
      },
      Q = {
        messages = {
          "Quit",
        },
      },
    },
  },
}

xplr.config.modes.builtin.default.key_bindings.on_key["Z"] = {
  messages = {
    { SwitchModeCustom = "Z" },
  },
}

xplr.config.modes.builtin.default.key_bindings.on_key["enter"] = { messages = { "Enter" } }

xplr.config.modes.builtin.default.key_bindings.on_key["e"] = xplr.config.modes.builtin.action.key_bindings.on_key.e

xplr.config.modes.builtin.search.key_bindings.on_key.enter.messages = { "Enter", "PopMode" }

xplr.config.modes.builtin.search.key_bindings.on_key["/"] = { messages = { "Enter", "ResetInputBuffer" } }
```

**Reply by sayanarijit** [reply]
Same thing happen without config? Also, try another system (different pc/laptop). Something (on host/guest machine) might be capturing the key.

**Reply by ElSamhaa** [reply]
I've tried on a host and a VM, with and without custom config; same behavior. Also, I don't think it's an underlying issue, I think it's xplr behavior. pls see this recording


https://github.com/sayanarijit/xplr/assets/34912861/46b64316-4ed2-4f82-b5e4-453c37892663

**Reply by sayanarijit** [reply]
If a service in the host is capturing the key, it would capture the key in all the guest vms as well. Not to mention keyboard and driver issues.

Let's turn it into an issue, rather that discussion. Let's see if anyone else is having the same issue. It would be useful to single out the actual cause and reproducing it.

**Reply by sayanarijit** [reply]
https://github.com/sayanarijit/xplr/issues/685
